### DISCUSSION TITLE
Dark mode (theme): availability and if so, how to enable?

### DISCUSSION THREAD

**Discussion by wechsler42** [bpatrik/pigallery2#542] [Q&A]
Hi there,
reading in the open and closed issues it seems that there might be a dark mode (theme) available in PiGallery2. However, I am not sure on this and I do not know where I could enable it. Perhaps someone would be that kind to enlighten me on this?
Thanks in advance .

**Comment by bpatrik** [comment]
There is no dark mode available at the moment.

**Comment by bpatrik** [comment]
This is now implemented (not yet part of a release , only the nightly build sofar)
